app/lib/common/models/medication.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:convert';
+
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:http/http.dart';
+
+import 'guideline.dart';
+
+part 'medication.freezed.dart';
+part 'medication.g.dart';
+
+@freezed
+class Medication with _$Medication {
+  const factory Medication(int id, String name, String description,
```

Comment:
just trying to understand the practical difference between Medication and MedicationWithGuidelines: Why does Medication not have a rxcui and pharmgkbId?